python/orion_finance_sdk_py/contracts.py

```
@@ -458,23 +500,75 @@ def get_price(self, asset: str, block: int | None = None) -> int:
+            assets: Optional token addresses to price. Defaults to the full
+                whitelisted investment universe from ``OrionConfig``.
+
+        Returns:
+            List of ``{"timestamp", "block", "prices"}`` dicts where ``prices``
+            maps checksummed asset address to price (scaled by
+            ``price_adapter_decimals``).
+        """
+        if interval != "1d":
+            raise ValueError(
+                f"Unsupported interval {interval!r}. Only '1d' is supported."
+            )
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)